src/lib/apiClient.ts

```
@@ -0,0 +1,25 @@
+  timeout: 10000,
+})
+
+// Attach token from cookies if available
+apiClient.interceptors.request.use((config) => {
+  const tokenCookie = Cookies.get('thisisjustarandomstring')
+  if (tokenCookie) {
+    try {
+      const token = JSON.parse(tokenCookie)
+      config.headers.Authorization = `Bearer ${token}`
+    } catch (error) {
+      console.error('Error parsing auth token from cookie', error)
```

ESLint

[error] Unexpected console statement.

(no-console)